src/app/modules/group/pages/group-details/group-details.component.html

```diff
@@ -84,10 +84,18 @@
   <alg-error
```

Comment:
![image](https://user-images.githubusercontent.com/16257159/148202488-c964f6ea-b998-4c21-b7ba-68607ea05f0a.png)
The reload button is a nice touch for "Error while loading group info" but I don't see the point of reloading when you are not allowed to see the content

Reply:
Fixed

Reply:
Now it's also removed for "Error while loading group info", isn't there a way to have the refresh button for "Error while loading group info" but not "You are not allowed to view this group page" ?

Reply:
Fixed